packages/superset-ui-preset-chart-xy/src/Line/buildQuery.ts

```diff
@@ -0,0 +1,6 @@
+import { buildQueryContext } from '@superset-ui/chart';
+import ChartFormData from './ChartFormData';
+
+export default function buildQuery(formData: ChartFormData) {
+  return buildQueryContext(formData);
```

Comment:
Note that `buildQueryContext` cannot handle the time shift functions.

Reply:
https://github.com/apache/incubator-superset/blob/master/superset/viz.py#L1194

Reply:
It's ok. this will not handle time shift yet